[DOCS] Update contributing code readme with required markers information. (#8414)
great-expectations/great_expectations: CONTRIBUTING_CODE.md

Testing can generate warning messages. These warnings are often caused by dependencies such as pandas or SQLAlchemy. Run `pytest --no-sqlalchemy --disable-pytest-warnings` to suppress these warnings.

### Marking tests

> All tests in Great Expectations are required to be marked with one marker from the list `REQUIRED_MARKERS` defined in [tests/conftest.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/conftest.py).

Reviewer: ```suggestion
All tests in Great Expectations must include one marker from the `REQUIRED_MARKERS` list. To view the list of defined markers, see [tests/conftest.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/conftest.py).
```

> One can verify each test is marked by running `invoke marker-coverage` if [invoke](https://pypi.org/project/invoke/) is installed or by running `pytest --verify-marker-coverage-and-exit`.

Reviewer: ```suggestion
To verify each test is marked, run `invoke marker-coverage` if [invoke](https://pypi.org/project/invoke/) is installed, or run `pytest --verify-marker-coverage-and-exit`.
```

> If the verification fails, the list of unmarked tests and the required markers will be outputted.

Reviewer: ```suggestion
When verification fails, a list of unmarked tests and the required markers appears.
```
